### DISCUSSION TITLE
SCM output timestep

### DISCUSSION THREAD

**Discussion by kashtanka** [ARM-Development/comble-mip#50] [General]
Hello,
I just want to make sure if I understand correctly.
We are requested to upload SCM results averaged over last 10 minutes of each hour. Is that right? The operational time step of our model is 270 s, which is 4:30 min. So we can either average over the last 9 minutes of an hour, or change the time step to better fit into 10 min.  
Also, the output of the E3 model has 30 min time step, not one hour. Should we also use 30 min?
Which way to proceed would you suggest? 
Best,
Dmitry

**Comment by fridlind** [comment]
Thank for this question! Please report the time-average over your SCM model time step, using whichever time step you prefer to run the SCM for this intercomparison. That was an inadvertent oversight to leave the "10-min" average from the LES template; we have corrected the output header to indicate the SCM time step average.

**Reply by AlejandroBaroPerez** [reply]
Hi @fridlind : 
Just to be sure, should the Mean outputs be "10 minutes instantaneous" or "10 minutes average" in the LES models? 
I see that it is written now:
" comment (SCM time-step average reported at endpoints, green=minimum) "  So, there is no clear specification for the LES models...

Cheers,
Alejandro

**Reply by fridlind** [reply]
Thank you for drawing attention to this. I got confused myself by the previous question about SCM output. Here is the request:

(1) LES output as 10-min averages
(2) SCM output as time-step averages (using operational time step)

If the headers are not clear on each page now, please suggest wording that would be clearer. Many thanks!

**Reply by kashtanka** [reply]
Thank you for the clarification!
